Gemfile

```diff
@@ -147,3 +147,4 @@ gem 'dog_biscuits', git: 'https://github.com/samvera-labs/dog_biscuits.git'
 gem 'order_already'
 
 gem 'hyrax-v2_graph_indexer'
+gem 'iiif_print', git: 'https://github.com/scientist-softserv/iiif_print.git', branch: 'main'
```

Comment:
doesn't block this but we should probably do a release

Reply:
https://github.com/scientist-softserv/iiif_print/issues/160